Source/Core/DolphinQt2/GameList/GameListModel.cpp

```diff
@@ -200,7 +200,7 @@ QSharedPointer<GameFile> GameListModel::GetGameFile(int index) const
   return m_games[index];
 }
 
-void GameListModel::UpdateGame(QSharedPointer<GameFile> game)
+void GameListModel::UpdateGame(const QSharedPointer<GameFile>& game)
```

Comment:
Maybe UpdateGame should just take a pointer directly? Passing a shared pointer by const ref seems weird.